app/javascript/dashboard/components/ConversationItem.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <conversation-card
+    :key="source.id"
+    :active-label="label"
+    :team-id="teamId"
+    :folders-id="foldersId"
+    :chat="source"
+    :conversation-type="conversationType"
+    :selected="isConversationSelected(source.id)"
+    :show-assignee="showAssignee"
+    :enable-context-menu="true"
+    @select-conversation="onSelectConversation"
+    @de-select-conversation="onDeSelectConversation"
+    @assign-agent="onAssignAgent"
+    @assign-team="onAssignTeam"
+    @assign-label="onAssignLabel"
+    @update-conversation-status="toggleConversationStatus"
+    @context-menu-toggle="onContextMenuToggle"
+    @mark-as-unread="onClickMarkAsUnread"
+    @assign-priority="onAssignPriority"
```

Comment:
Does it work if we pass the injected methods here directly?

Reply:
> Does it work if we pass the injected methods here directly?

I think it will work. But here this component is used in 2 places. In chatlist and previous conversation list.

Reply:
That also works. Done https://github.com/chatwoot/chatwoot/pull/8540/commits/55d56c88bfe8fad76d7b6db5251fced1c66a8871